src/cli.rs

```diff
@@ -643,6 +643,7 @@ pub enum FileType {
     Directory,
     #[value(alias = "l")]
     Symlink,
+    /// Filter file which are executable by the current effective user
```

Comment:
This is not the filter itself. Maybe just
```suggestion
    /// A files which is executable by the current effective user
```

Also, what is the current *effective* user? Is that a known term? Or could it simply be
```suggestion
    /// A file which is executable by the current user
```

Reply:
I agree, the filter part is not relevant, as for the term "effective user" it came from the doc at [faccess](https://docs.rs/faccess/0.2.4/faccess/trait.PathExt.html#tymethod.access)
From a quick search it appear that:
* effective user: is the user which call the executable, which can be root in case of non-privileged user using `sudo`
* RealId user: user which is the original user who have called the executable

Reply:
The effective and real users are only different for setuid executables themselves.  Presumably no one makes `fd` a setuid executable, so they will be the same.

Reply:
I think the effective and real user could also be different if a program running as root set the effective user to something different, then execed fd with the right flags or environment. But that also seems contrived. I'm fine with just saying current user.

Reply:
Thank you. In that case, I'm fine with either option.